src/TTextEdit.cpp

```diff
@@ -1390,6 +1390,7 @@ QString TTextEdit::getSelectedText(char newlineChar)
         // we never append the last character of a buffer line se we set our own
         text.append(newlineChar);
     }
+    return text;
```

Comment:
Yeah, I had send this in the clang compiler warning output but I do not grok this enough to say whether it is actually reachable in real life.  (I suspect Codecy will have something to say about if it can be reached and exactly how that occurs...)

Reply:
... perhaps a `qDebug()` to report that this point has been hit may be worthwhile?